Jellyfin.Data/Entities/MediaSegment.cs

```diff
@@ -0,0 +1,62 @@
+#nullable disable
+#pragma warning disable CS1591
+
```

Comment:
New files shouldn't be disabling NRF or warnings (Especially DB entities, we use NRF to indicate required vs nullable fields). It looks like everything is already documented so you can just remove the #pragma